src/App.native.tsx

```diff
@@ -97,7 +99,7 @@ function InnerApp() {
     <SafeAreaProvider initialMetrics={initialWindowMetrics}>
       <Alf theme={theme}>
         <ThemeProvider theme={theme}>
-          <Splash isReady={isReady}>
+          <Splash isReady={isReady && hasCheckedReferrer}>
```

Comment:
Flagging as a concern that we need to make sure `hasCheckedReferrer` _definitely_ resolves to `true` in reasonable time (no matter what errors etc) because otherwise we'll get stuck on the splash screen which would be Very Bad.